docs/README.md

```diff
@@ -1,28 +1,152 @@
+securely consumes a managed `MySQLInstance` from GCP, AWS, or Azure from
+`kubectl`.
+
+## Crossplane Stacks
+Stacks extend Crossplane with new functionality and can be
+installed using the [Stack
+Manager](https://github.com/crossplaneio/crossplane/blob/master/design/design-doc-stacks.md)
+via the [Kubernetes
+API](https://github.com/crossplaneio/crossplane/blob/master/design/design-doc-stacks.md#installation-flow)
+or with the [crossplane kubectl
+plugin](https://github.com/crossplaneio/crossplane-cli#crossplane-cli) [`stack
+install`](https://github.com/crossplaneio/crossplane-cli#usage) command.
+Crossplane is built on the Kubernetes API machinery as a platform, and Stacks
+simplify extending the Kubernetes API to provision and lifecycle manage apps and
+the managed service infrastructure they depend on, including [Stack security and
+isolation](https://github.com/crossplaneio/crossplane/blob/master/design/one-pager-stacks-security-isolation.md).
+
+The [Using Stacks](using-stacks.md) guide shows how to deploy a [portable
+Wordpress App Stack](https://hub.docker.com/r/crossplane/sample-stack-wordpress)
+into multiple clouds using Infra Stacks including
+[stack-gcp](https://github.com/crossplaneio/stack-gcp),
+[stack-aws](https://github.com/crossplaneio/stack-aws), and
+[stack-azure](https://github.com/crossplaneio/stack-azure). 
+
+### Infrastructure Stacks 
+Infra Stacks like
+[stack-gcp](https://github.com/crossplaneio/stack-gcp),
+[stack-aws](https://github.com/crossplaneio/stack-aws), and
+[stack-azure](https://github.com/crossplaneio/stack-azure) extend the Kubernetes
+API to support managed service provisioning (DBaaS, cache, buckets), secure
+connectivity (VPCs, subnets, peering, ACLs, secrets), and provisioning managed
+Kubernetes clusters on demand to further isolate the blast radius of
+applications. 
+
+Infra Stacks are pre-built and published to the [Stacks
+registry](https://hub.docker.com/search?q=crossplane&type=image) where they can
+be installed by a Cluster Admin using a
+[`ClusterStackInstall`](https://github.com/crossplaneio/crossplane/blob/master/design/design-doc-stacks.md#installation-flow)
+Kubernetes API kind or with the [`stack
+install`](https://github.com/crossplaneio/crossplane-cli#usage) command that
+installs a Stacks with permissions to watch resources at a cluster level, across
+all namespaces.
 
-The full architecture and vision of the Crossplane project is described in depth in the [architecture document](https://docs.google.com/document/d/1whncqdUeU2cATGEJhHvzXWC9xdK29Er45NJeoemxebo/edit?usp=sharing). It is the best place to learn more about how Crossplane fits into the Kubernetes ecosystem, the intended use cases, and comparisons to existing projects.
+Infra Stacks are covered in both the [Using Services](using-services.md) and
+[Using Stacks](using-stacks.md) guides. 
+
+### Application Stacks 
+App Stacks depend on Infra Stacks like
+[stack-gcp](https://github.com/crossplaneio/stack-gcp),
```

Comment:
nitpick: For references, it would be easier to use a reference like below at the end of the document, and then just use `[stack-gcp]` throughout the document.
```
[stack-gcp]: https://github.com/crossplaneio/stack-gcp
```

Reply:
fixed